### PR TITLE
fix(CI): Install composer dependencies of Talk

### PR DESCRIPTION
**Pull request by nickvergessen**
* Resolves: N/A

## Summary

Ref https://github.com/nextcloud/spreed/pull/8578
Found in https://github.com/nextcloud/server/pull/36310#issuecomment-1405068637

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [x] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by ChristophWurst — APPROVED** [review]
:pray:

**Review by danxuliu — CHANGES_REQUESTED** [review]
[CI is not happy](https://drone.nextcloud.com/nextcloud/server/29226/45/3):
```
+ composer install --no-dev
/bin/sh: composer: not found
```

File: .drone.yml [hunk 1]
```diff
@@ -1236,7 +1236,7 @@ steps:
   commands:
     - git submodule update --init
 - name: install-talk
-  image: ghcr.io/nextcloud/continuous-integration-integration-php8.0:latest
+  image: ghcr.io/nextcloud/continuous-integration-php8.0:latest
```
Comment (vitormattos) [inline .drone.yml:1239]: Have a problem during execution of `composer install`:

```
  - bamarni/composer-bin-plugin 1.8.2 requires composer-plugin-api ^2.0 -> no matching package found.
```

The composer version of this image is very old:

https://github.com/nextcloud/docker-ci/blob/master/php8.0/Dockerfile#L18

**Comment by ChristophWurst** [comment]
you can also use `ci` as prefix for conventional commits. tools that generate changelogs would then hide the commit. a fix would be listed as a fix on the product. end of nitpick.

**Comment by nickvergessen** [comment]
So, we are green and good again :)

**Review by blizzz — APPROVED** [review]

**Pull request merged**